spec/controllers/course_summaries_controller_spec.rb

```diff
@@ -248,20 +240,6 @@
           }
           expect(@data).to include expected
         end
-
-        it 'returns no info about assessments' do
```

Comment:
Maybe I'm just not seeing how you adapted this test elsewhere, but I think the purpose of this test is still good and worth checking (e.g. by checking `@response_data[:average_data]` etc.)